app/presenters/tree_builder_dialog_edit.rb

```diff
@@ -0,0 +1,70 @@
+class TreeBuilderDialogEdit < TreeBuilder
+  has_kids_for Hash, [:x_get_tree_hash_kids]
+
+  def initialize(name, type, sandbox, build = true, root = nil)
+    @root = root
+    super(name, type, sandbox, build)
+  end
+
+  private
+
+  def tree_init_options(_tree_name)
+    {:full_ids => true, :lazy => false, :open_all => true}
+  end
+
+  def set_locals_for_render
+    locals = super
+    locals.merge!(:id_prefix                   => "de_",
+                  :autoload                    => true,
+                  :url                         => '/vm/show/',
+                  :open_close_all_on_dbl_click => true,
+                  :cookie_id_prefix            => "edit_treeOpenStatex",
+                  :exp_tree                    =>  true,
+                  :onclick                     => "miqOnClickSelectDlgEditTreeNode",)
```

Comment:
You'll need to add this function to `miqTreeEventSafeEval` after rebasing, otherwise it'll stop working ;) (#9943)